katran/lib/KatranLbStructs.h

```diff
@@ -59,6 +59,11 @@ struct RealMeta {
    * only when refcount would be equal to zero
    */
   uint32_t refCount;
+
+  /**
+   * real flags
```

Comment:
i don't think this is need as it does not gave any aditional information, which can not be deduced from variable's name